pkg/frinkiac/http/client_test.go

```diff
@@ -49,7 +61,7 @@ func ExampleGetScreenCap() {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
 
-	result, err := GetScreenCap(ctx, client, config, "S09", "E22", "202334")
+	result, err := GetScreenCap(ctx, client, config, "S09", "E22", 202334)
```

Comment:
Introduce a type definition from the timestamp string.  Make it a string.  Update the Timestamp field in the APISearchResults struct and the input parameter for GetScreenCap